chapter_2/component_interface/CMakeLists.txt

```diff
@@ -3,10 +3,26 @@
 # SPDX-License-Identifier: MIT
 
 project(
-  ch2.framework.component_interface
-  VERSION 1.0
-  DESCRIPTION "Chapter 2 component interface implementation"
-  LANGUAGES CXX)
+    ch2.framework.component_interface
+    VERSION 1.0
+    DESCRIPTION "Chapter 2 component interface implementation"
+    LANGUAGES CXX
+)
 
 add_library(${PROJECT_NAME} INTERFACE)
-target_include_directories(${PROJECT_NAME} INTERFACE include/)
+target_include_directories(
+    ${PROJECT_NAME} INTERFACE include/ $<INSTALL_INTERFACE:include>
```

Comment:
This genex is not needed here or in general. The code doesn't handle the creation of relocatable CMake packages yet, so there is no install interface where this could go.

Reply:
Removed it in last revision.